MailCore/Cache/AccountManager/AccountManager.swift

```diff
@@ -32,11 +32,12 @@ import SwiftUI
 public extension InfomaniakNetworkLoginable {
     func apiToken(using code: String, codeVerifier: String) async throws -> ApiToken {
         try await withCheckedThrowingContinuation { continuation in
-            getApiTokenUsing(code: code, codeVerifier: codeVerifier) { token, error in
-                if let token {
+            getApiTokenUsing(code: code, codeVerifier: codeVerifier) { result in
+                switch result {
+                case .success(let token):
                     continuation.resume(returning: token)
-                } else {
-                    continuation.resume(throwing: error ?? MailError.unknownError)
+                case .failure(let error):
+                    continuation.resume(throwing: error)
                 }
             }
```

Comment:
We should now be able to use the async await version of this function provided by login